src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/PreviewControllerTest.php

```diff
@@ -119,7 +119,6 @@ public function testRender()
         $client->request('POST', '/api/nodes?&webspace=sulu_io&language=en', $data);
         $response = json_decode($client->getResponse()->getContent());
 
-        $client->request('GET', '/content/preview/' . $response->id . '/start?webspace=sulu_io&language=en');
```

Comment:
It probably returned a 404, but there was no check on anything for this request.